akka-docs/src/main/paradox/scala/cluster-usage.md

```diff
@@ -111,8 +108,11 @@ The source code of this sample can be found in the
 ## Joining to Seed Nodes
```

Comment:
@WadeWaldron I tried to incorporate some of your findings in the documentation of the seed nodes. Please review, and let me know if there is anything more or if still unclear.